docs/commands/SetParameters.md

```diff
@@ -0,0 +1,81 @@
+# SetParameters
+
+Parameters setup command. The sensor confirms the execution of the command with [response](#response)
```

Comment:
```suggestion
Parameters setup command.

The sensor confirms the execution of the command with [response](#response)
```

but should we really add the second line?

Reply:
who knows the answer?